src/main/java/com/chargerrobotics/subsystems/LimelightSubsystem.java

```diff
@@ -50,11 +51,33 @@ public void setLEDStatus(boolean enabled) {
 		camMode.setNumber(enabled ? 0 : 1);
 	}
 
+	// Placeholder values for coordinates of markers, in inches (should probably put
+	// these in constants but don't want to screw up anything)
+	// Looking down on the field with the initiation line to the south, the markers  
+	// would be placed as:
+	// 2|                 N                 |3
+	//
+	// W            \(r😏bot)/              E
+	//
+	//
+	// 1| __________________________________|4
+	//                   S
+	//The lines next to each number represent the way a board covered in retroreflective tape of a specfic color or shape are placed in respect to eachother.
+
+	public final double onex = 0;
```

Comment:
Since these are all constant values, a nice way to denote this is to make them all caps. For instance,

```java
public final double ONE_X = 0;
public final double ONE_Y = 0;
...
```

Though these variable names don't readily describe what they mean elsewhere, perhaps something like `MARKER_ONE_X`.